### PR TITLE
[Doc] Add difficulties to discussion

### LINKED ISSUE #190: Move the screenshots to appendix
It is not very important. We might as well drop it.

### LINKED ISSUE #191: Elaborate the discussion section
The current discussion section is not informative. We should elaborate it with difficulty (in technology, in collaboration, in scheduling).

### PR DESCRIPTION
**Pull request by Huy-Ngo**
I add the difficulties in doing the group project to the discussion.
If you find yourselves to have some other difficulties feel free to add.

Resolve #191
Also close #190 because I deleted the images.

### CODE REVIEW

**Comment by Huy-Ngo** [comment]
Should I delete the screenshot themselves too?

File: docs/source/result.rst [hunk 1]
```diff
@@ -76,15 +33,38 @@ Nonetheless, there are numerous flaws that we have found in our system:
...
+
+During the project, we met several difficulties in different aspects.
+
+The primary obstacle for us was the lack of time spent for the project.
+This lack of time dedicated to the project was because of poor time management.
+Moreover, there had been other time-consuming projects and labworks in other courses,
+which make the management harder.
+
+Another problem was the collaboration: There were several conflicts among
+some members' expectations of the project and workflows.  People tended to work
```
Comment (9a24f0) [inline docs/source/result.rst:56]: Double space here

File: docs/source/result.rst [hunk 2]
```diff
@@ -76,15 +33,38 @@ Nonetheless, there are numerous flaws that we have found in our system:
...
+at different time in day, which made code reviews take a longer time than it should.
+Some members were not too keen on the project and did not spend enough time for it.
+
+There were also some technical struggles: Not everyone in the group was familiar
+with web development in Python, let alone the framework.  This revealed another
```
Comment (9a24f0) [inline docs/source/result.rst:61]: Here too

**Review by 9a24f0 — CHANGES_REQUESTED** [review]
I'm good with the difficulties, you should also remove the screenshots too.

**Review by 9a24f0 — APPROVED** [review]
LGTM!

**Pull request merged**